scripts/typescript-compatibility-build.yml

```diff
@@ -123,7 +123,7 @@ jobs:
 
       # Build sample app consuming Office UI Fabric React to test TypeScript version compatibility
       - script: |
-          node common/scripts/install-run-rush.js test --to $(TypeScriptVersion)-office-ui-fabric-react-test
+          node common/scripts/install-run-rush.js test --to $(TypeScriptVersion)-fluentui-react-native-test
```

Comment:
Do we actually run this compatability build pipeline anywhere? It's probably a good thing to think about, but if we're not using it anywhere let's maybe get rid of it?

Reply:
Let's add a backlog item to do so in our build-maintenance efforts; I don't want to add it to this change